src/Schema/ForeignKeyConstraintEditor.php

```diff
@@ -60,13 +76,73 @@ public function setReferencingColumnNames(
         return $this;
     }
 
+    /**
+     * @param non-empty-string $firstColumName
+     * @param non-empty-string ...$otherColumnNames
+     */
+    public function setUnquotedReferencingColumnNames(
+        string $firstColumName,
+        string ...$otherColumnNames,
+    ): self {
+        $this->referencingColumnNames = array_map(
+            static fn (string $name): UnqualifiedName => UnqualifiedName::unquoted($name),
+            array_merge([$firstColumName], array_values($otherColumnNames)),
```

Comment:
Why not use unpacking here?

```suggestion
            [$firstColumnName, ...$otherColumnNames]
```

Reply:
I tried it some time ago the way you suggested, and PHPStan was unhappy for some reason. I switched to `array_merge()`, it still was unhappy, and then I realized that PHP would use the keys from `$otherColumnNames` as named arguments as a result of `...`. I added `array_values()` and forgot to switch back to unpacking.